WordPress/Classes/ViewRelated/Stats/Period Stats/SiteStatsPeriodTableViewController.swift

```diff
@@ -183,7 +194,17 @@ private extension SiteStatsPeriodTableViewController {
             return
         }
 
+        DDLogInfo("--- Refresh Table View")
```

Comment:
Is this necessary? It's creating a bit of log noise, and doesn't really provide any information...

Reply:
Yes I need it to actually verifying the table is reloaded immediately after an operation is executed.
I'm going to remove it here since I have it in another branch 👍